README.Rmd

```diff
@@ -64,43 +64,37 @@ This repository shows off a really exciting set of capabilities, combining open-
 - Jobs don't depend on another. I've scheduled the jobs so that each will complete by the time another starts, but there are tools in R (like [drake](https://github.com/ropensci/drake)) that allow you to put the entire pipeline into code and make dependencies explicit.
 - Pieces of content must be managed individually -- that includes uploading, permissions, environment variables, and tagging. It is possible to do something more robust via programmatic deployment using the RStudio Connect API, but generic git deployment doesn't support deploying all of the content in a git repo at once.
 
-# Content in this App
-
-Here's what the system looks like 
-![](./img/system_schematic.png)
```

Comment:
Any reason to drop this entirely? It's much simpler now...

Reply:
I mostly didn't want to recreate the png with the correct schematic ..... 

I did keep the high level schematic at the top which I regularly use to introduce the moving parts, but I typically handwave a bit about what the specific moving parts are.

Reply:
Lol, fair. That slide was a pain to make. I can't even find it now to edit...